release/kata-deploy-binaries.sh

```diff
@@ -167,6 +167,18 @@ install_firecracker() {
 	popd
 }
 
+# Install static cloud-hypervisor asset
+install_cloud_hypervisor() {
```

Comment:
@jcvenegas nit: Can this be renamed `install_clh`, because I assumed that name in our github actions.
https://github.com/kata-containers/kata-containers/pull/58/commits/9ae852f438fd989507b8e13b9e82789e455f86d3#diff-295c250d2ddc435b0eafc807f0b3cac9R192

We need to change it in one or the other place depending on what function name we like better:)

Reply:
Lets keep install_clh